code/modules/clothing/gloves/color.dm

```diff
@@ -85,6 +85,14 @@
 	. = ..()
 	siemens_coefficient = pick(0,0.5,0.5,0.5,0.5,0.75,1.5)
 
+	if(Holiday == APRIL_FOOLS_DAY && !siemens_coefficient == 0)
```

Comment:
why are you not using != here?
!siemens_coefficient == 0 is a bit ambiguous since you could reasonably read it as logical not to the siemens_coefficient, then check if it is 0. (the end result is the same, which is the most cursed of all)

Reply:
the worst part is that the code still works how (i assume) it was intended to

Reply:
i forgot about != lel, i can change it

Reply:
```suggestion
	if(Holiday == APRIL_FOOLS_DAY && siemens_coefficient != 0)
```